core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -625,7 +654,7 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
         domain: this.service.domain,
         service: this.service.name,
         environment: this.filters.getFieldValueOfLabel('ENVIRONMENT') || this.activatedRoute.snapshot.params['env'] || 'prod',
-        start_time: this.filters.getFieldValueOfLabel('TIME RANGE').range,
+        start_time: this.startTime,
         end_time: endDateTime,
```

Comment:
The `endDatetime` should just be the `currentTime` i.e. `moment()` and `startTime` should be calculated based on `endDateTime`. For most part it is ok now, but is off by a few milliseconds because of how it is implemented.